quizbizz/src/pages/authentication/login/Loginform.jsx

```diff
@@ -30,59 +32,55 @@ const Loginform = () => {
   };
   const loginUser = async () => {
```

Comment:
Suggestion: you should move this function to a separate file

Reply:
OK